scripts/run.sh

```diff
@@ -2,4 +2,4 @@
 
 killall sifnoded
 
-sifnoded start --trace
+sifnoded start --trace --home ~/.sifnoded/ --p2p.laddr 0.0.0.0:27655  --grpc.address 0.0.0.0:9096 --grpc-web.address 0.0.0.0:9093 --address tcp://0.0.0.0:27659 --rpc.laddr tcp://127.0.0.1:26657
```

Comment:
Added explicit parameters to manage the script better, and use it while running multiple networks while testing 
Default ports are used for the ports used for broadcasting in our test scripts.